scripts/libunwind_build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -euo pipefail
+
+curl -L http://download.savannah.nongnu.org/releases/libunwind/libunwind-1.5.0.tar.gz -o libunwind-1.5.0.tar.gz
```

Comment:
Can be https o.O

Reply:
Whoops. I copy-pasted it from somewhere in the code and didn't noticed

Reply:
Yeah it was this way from earlier haha. Let's improve